.travis.yml

```diff
@@ -139,14 +139,3 @@ script:
   - ln -s install-libspatialite-4.0dev/lib/libspatialite.so.5.0.1 libspatialite.so.3
   - LD_LIBRARY_PATH=$PWD python ogr_sqlite.py
   - cd ..
-
-notifications:
-  email:
-    recipients:
-      - gdal-commits@lists.osgeo.org
-
-  irc:
-    channels:
-      - "irc.freenode.org#gdal"
-    use_notice: true
-    on_success: change
```

Comment:
It will be restored after everything will be ready to merge (since Even asked to disable IRC notifications on failed builds).